Neos.Fusion/Tests/Unit/FusionObjects/DataStructureImplementationTest.php

```diff
@@ -14,118 +14,113 @@
 use Neos\Flow\Tests\UnitTestCase;
 use Neos\Fusion\Core\Runtime;
 use Neos\Fusion\FusionObjects\DataStructureImplementation;
+use PHPUnit\Framework\MockObject\MockObject;
 
 /**
  * Testcase for the Fusion Concat object
  */
 class DataStructureImplementationTest extends UnitTestCase
 {
+    /**
+     * @var Runtime|MockObject
+     */
+    private $mockRuntime;
+
+
+    public function setUp(): void
+    {
+        $this->mockRuntime = $this->getMockBuilder(Runtime::class)->disableOriginalConstructor()->getMock();
+    }
+
     /**
      * @test
      */
-    public function evaluateWithEmptyArrayRendersEmptyArray()
+    public function evaluateWithEmptyArrayRendersEmptyArray(): void
     {
-        $mockRuntime = $this->getMockBuilder(Runtime::class)->disableOriginalConstructor()->getMock();
-        $path = 'array/test';
+        $path = 'datastructure/test';
         $fusionObjectName = 'Neos.Fusion:DataStructure';
-        $renderer = new DataStructureImplementation($mockRuntime, $path, $fusionObjectName);
+        $renderer = new DataStructureImplementation($this->mockRuntime, $path, $fusionObjectName);
         $result = $renderer->evaluate();
         self::assertSame($result, []);
     }
 
     /**
      * @return array
      */
-    public function positionalSubElements()
+    public function positionalSubElements(): array
     {
+        $ds = '<Neos.Fusion:DataStructure>';
         return [
             [
                 'Position end should put element to end',
-                ['second' => ['__meta' => ['position' => 'end']], 'first' => ['__meta' => []]],
-                ['/first', '/second']
+                ['second' => ['__meta' => ['position' => 'end']], 'first' => []],
```

Comment:
before this fix having a `__meta` key like `@if` or `@position` would lead to the key `second` not being identified as untyped.

now it is and we need to adjust these test.